gui/LineEditor.cxx

```
@@ -0,0 +1,237 @@
+#include <TGTextEntry.h>
+#include <TGWidget.h>
+#include <TGedFrame.h>
+#include <TGraph.h>
+#include <TMathBase.h>
+#include <TROOT.h>
+
+namespace Bober {
+  LineEditor::LineEditor(TGCompositeFrame* frame, TVirtualPad* pad) : MainEditor(frame, pad) {
+    enum ELineWid { kCOLOR, kLINE_WIDTH, kLINE_STYLE, kALPHA, kALPHAFIELD };
+
+    fPriority = 1;
```

cppcheck (2.10-2)

[performance] Variable 'fPriority' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)